content/en/docs/Configuration/customization.md

```diff
@@ -9,39 +9,38 @@ date: 2021-09-02
 
 ## Solution Branding
 
-You can easily apply your branding on most solution user interface.
-This is usually change logos and style (color palette, font, etc.)
+You can easily apply your branding on most of the solution's user interface, such as changing the logos and the style (color palette, font, and so on.)
 
-Here are the main items you can consider for branding:
+The following are the main items you can consider for branding:
 
-* Logo and style in Solution Homepage: [API Gateway > Solution Homepage](/docs/configuration/api-gateway#solution-homepage)
+* Solution Homepage logo and style: [API Gateway - Solution Homepage](/docs/configuration/api-gateway#solution-homepage).
     ![Welcome email](/Images/homepage.png)
-* Logo and style in Email templates: [API manager > Email templates override](/docs/configuration/api-manager#email-templates-override)
+* Email templates logo and style: [API manager - Email templates override](/docs/configuration/api-manager#email-templates-override).
     ![Welcome email](/Images/welcome-email.png)
-* Logo and style of Developer Portal: [Developer portal > Template customization](/docs/configuration/portal/#template-customization)
-* Logo and style of Analytics homepage. Refer to deployment option for this component : [Customize Analytics Helm chart](/docs/deployment/installation/analytics/#customize-analytics-helm-chart)
+* Developer Portal logo and style: [Developer portal - Template customization](/docs/configuration/portal/#template-customization).
+* Analytics homepage logo and style. Refer to the deployment option for this component: [Customize Analytics Helm chart](/docs/deployment/installation/analytics/#customize-analytics-helm-chart).
 
 ## Consent page
 
-Each Axway Open Banking customer would need to customize the Consent page that is embedded in the solution.
+Each Axway Open Banking customer needs to customize the Consent page that is embedded in the solution.
```

Comment:
The Consent page embedded in the solution is provided for demonstration purposes only. Each customer should customize the consent page to match their target application's requirements.